doc/CONTRIBUTING.md

```diff
@@ -3,9 +3,27 @@
 # Rubygem doc:
 http://www.rubydoc.info/gems/gitarro/
 
-
 # How to contribute to gitarro development
 
+
+## Issues before big Prs aka features 
+
+If you want to contribute, just look at current issues and start to discuss/thinking how you will solve some of them.
+If you have in mind a new feature, just **open an issue first.**
+ 
+Instead of Proposing directly a Pull Request, openining an issue where you describe what you want to do, how, etc.,
+will give you directly a feedeback, and have a sense of knowing what you will do by describing, discussing it **before**, 
+will  increase the ease you will code ( discussing the design before, instead of changing it after).
+
+
+For small bugfixing/improvements of course you don't need an Issue referenced.
+
```

Comment:
And between sections (in the end GitHub will only consider one line when using markdown)